.size-snapshot.json

```diff
@@ -0,0 +1,36 @@
+{
+  "./dist/formik.umd.production.js": {
+    "bundled": 180899,
+    "minified": 48286,
+    "gzipped": 14197
```

Comment:
Do we have to commit this?

Reply:
Yes. This is the point. Every user can take a look at snapshot and get info about size and treeshakability.